lib/paypal_express/api.rb

```diff
@@ -301,6 +301,29 @@ def get_payer_info(token,
         response
       end
 
+      def create_billing_aggrement(token,
+                                   kb_account_id,
+                                   kb_tenant_id,
+                                   payment_processor_account_id,
+                                   kb_payment_id = nil,
+                                   kb_payment_transaction_id = nil,
+                                   transaction_type = nil)
+        logger.info("CreateBillingRecord")
+        raise 'Could create billing aggrement: the token is missing' if token.blank?
```

Comment:
Should we not check for `nil` `token` ?

Reply:
`blank?` will check only for empty string.

Reply:
we have checked token value for nil during function call. we are doing defencing checking for empty string

Reply:
Got it.